### DISCUSSION TITLE
Python package install error

### DISCUSSION THREAD

**Discussion by LucasSaito930903** [community/community#132480] [Packages] [answered]
### Select Topic Area

Question

### Body

While I am installing python package, an error occurred related cmaker 14

**Comment by Gambo019** [comment]
Which version of python are you using?

**Comment by LucasSaito930903** [comment]
Windows installer 3.12 64bit

**Comment by Gambo019** [comment, marked as answer]
Maybe, that error occurred due to version. Version you are using is not stable for packages except built-in ones.
Plz use new version 3.10.10
